src/libraries/System.Private.CoreLib/src/System/Environment.Windows.cs

```diff
@@ -86,7 +87,25 @@ private static string ExpandEnvironmentVariablesCore(string name)
             Interop.Kernel32.GetComputerName() ??
             throw new InvalidOperationException(SR.InvalidOperation_ComputerName);
 
-        private static int GetCurrentProcessId() => unchecked((int)Interop.Kernel32.GetCurrentProcessId());
+        [MethodImplAttribute(MethodImplOptions.NoInlining)]
```

Comment:
This makes `Environment.ProcessId` about 3x faster by avoiding inlining PInvoke frame into the hot path.

Reply:
That might be a useful comment as it wouldn't have been obvoius to me.